lblod/ember-rdfa-editor-lblod-plugins
# Gn 4877

### Overview
Implemented Snippet fragments per Saska design, changed a little bit the text of the add new button, but otherwise the rest should be the same as the figma

##### connected issues and PRs:
GN-4877


### Setup
None

### How to test/reproduce
Insert a snippet placeholder, insert a snippet and then select it, you should see the buttons

### Challenges/uncertainties
I experienced some problems sometimes with clicking on the buttons, but then Elena couldn't reproduce them, so not sure if its my setup or not, looking for feedback on that so please test extensively



### Checks PR readiness
